src/components/Editor/ItemEditor.vue

```diff
@@ -164,34 +191,35 @@
         class="p-2"
         data-test="subjectiveQuestionContainer"
       >
-        <!-- checkbox -->
+        <!-- CharLimitSet checkbox -->
         <label class="inline-flex items-center mt-3">
           <input
             type="checkbox"
-            class="form-checkbox h-5 w-5 text-primary focus:ring-transparent"
+            style="box-shadow: none"
```

Comment:
can't you use a tailwind class for this?

Reply:
no doesn't work.
https://github.com/tailwindlabs/tailwindcss-forms/issues/10
see this - it's some issue with the tailwind forms plugin we're using.
It says he solved it using `ring-offset-0` but that doesn't work for some reason in our code. Adding `shadow-none` also doesn't work. Only overriding `style` works.